add expandable text with "see more/see less" in message bubble

Zeinab979 opened the pull request.

Zeinab979 committed:
add expandable text with "see more/see less" in message bubble

Zeinab979 committed:
Merge remote-tracking branch 'origin/develop' into refactor/refactor-message-bubble

Joseph-Sameh-0 commented on app/src/main/java/com/paris_2/san3a/presentation/screen/messagesDetails/components/MessageBubble.kt:
```diff
@@ -57,4 +59,93 @@ fun MessageBubble(
+                    contentPadding = PaddingValues(0.dp)
+                ) {
+                    Text(
+                        text = if (isExpanded) "See less" else "See more",
```
localize the "See less" and "See more"

Zeinab979 committed:
extract string resource

Zeinab979 committed:
Merge remote-tracking branch 'origin/develop' into refactor/refactor-message-bubble

Joseph-Sameh-0 committed:
Refactor: Remove unnecessary fillMaxWidth modifier

The `fillMaxWidth` modifier was removed from the `Column` in `MessageBubble.kt` as it was not needed for the component's layout.
